config/config_test.go

```diff
@@ -80,7 +81,8 @@ func TestGetUserConfigFromFile(t *testing.T) {
 }
 
 func TestGetUserConfigFromEnv(t *testing.T) {
-	t.Setenv("VULNBOT_DISABLE_SLACK", "1")
+	sliceAsStrings := strings.Join([]string{"slack"}, ",")
+	t.Setenv("VULNBOT_REPORTERS", sliceAsStrings)
```

Comment:
Instead of using `strings.Join()` and `strings.Split()` can you just use the explicit value here? Would it just be `"slack"` in this case?

Reply:
Fixed